src/components/layout/home/HomePage.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getCryptosList } from '../../../redux/cryptos/cryptos';
+import Crypto from '../../crypto/Crypto';
+import InputFilter from '../../filter/InputFilter';
+import Header from '../header/Header';
+import classes from './HomePage.module.css';
+
+const HomePage = () => {
```

Comment:
- [ ] The styling of the list on your Homepage does not follow the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), and therefore difficult to differentiate the details of each list item. Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that cells of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.


**Your design:**
![image](https://user-images.githubusercontent.com/12745474/142269200-e7cdfa53-04d5-488a-8af1-99f8c6df0c4d.png)



**The original design:**
![image](https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png)